src/components/index.ts

```diff
@@ -45,3 +45,4 @@ export { Search, SearchProps } from './Search/Search';
 export { Password } from './Password/Password';
 export { VisuallyHidden } from './VisuallyHidden/VisuallyHidden';
 export { RadioButtonProps } from './RadioButton/RadioButtonProps';
+export { Offcanvas, OffcanvasProps} from './Offcanvas/Offcanvas';
```

Comment:
I'm not sure where this is from now, but there's a space missing before the closing `}`. Wasn't this in another PR already?

Reply:
I think I have the problem that I can't pull from the wave remote so it's like my remote/branch thinks to be ahead then the wave one. 

I'm not sure why... I just fixed it here hope is not going to create conflicts in the other.